### PR TITLE
GPIO polling, add hardware.html, minor change to Gpio.cpp

### PR DESCRIPTION
**Pull request by hvbommel**
GPIO - Raspberry GPIO hardware gui update

### CODE REVIEW

File: www/views/hardware.html [hunk 1]
```diff
@@ -1165,20 +1165,29 @@ <h1 data-i18n="Settings">Settings</h1>
...
+					<td><input type="text" id="gpioperiod" style="width: 50px; padding: .2em;" class="text ui-widget-content ui-corner-all" value="50" />&nbsp;<span data-i18n="Milliseconds"></span>
 					<br />
 						<span data-i18n="Maximum of one interrupt per pin allowed in this period (prevents interrupt storms).">Maximum of one interrupt per pin allowed in this period (prevents interrupt storms).</span>
 						<br />
 					</td>
 				</tr>
+                <tr valign="top">
+                    <td align="right" style="width:110px"><label id="lblgpiopollinterval" for="name"><span data-i18n="Pollinterval"></span>Poll interval:</label></td>
+                    <td>
+                        <input type="text" id="gpiopollinterval" style="width: 50px; padding: .2em;" class="text ui-widget-content ui-corner-all" value="0" />&nbsp;<span data-i18n="Seconds"></span>
```
Comment (jvandenbroek) [inline www/views/hardware.html]: Better use` <span data-i18n="(Seconds) 0 = Disabled"></span>` and remove the line below, no extra explanation needed.

Comment (hvbommel) [reply www/views/hardware.html]: Done, thanks

**Comment by gizmocuz** [comment]
Thanks!

**Comment by gizmocuz** [comment]
Is it maybe possible that one of you would like to read the following forum topic ? Maybe you could help... It seems in the next kernel GPIO will not work anymore...

http://www.domoticz.com/forum/viewtopic.php?p=126756#p126756

I don't use it and am unable to assist here...

**Pull request merged**